stylesheets/_font_stack.scss

```diff
@@ -35,10 +27,7 @@ $NTA-Light-Tabular:
        local("Helvetica");
 }
 
-$Helvetica-Regular:
-  "GDS-Logo",
-  sans-serif;
+$Helvetica-Regular: "GDS-Logo", sans-serif;
```

Comment:
Should all the @font-face rules sit together? The rest are in the govuk template:
https://github.com/alphagov/govuk_template/blob/master/source/assets/stylesheets/fonts.css